packages/mdc-textfield/index.js

```diff
@@ -18,11 +18,12 @@
 import MDCComponent from '@material/base/component';
 import {MDCRipple} from '@material/ripple';
 
+/* eslint-disable no-unused-vars */
 import {cssClasses, strings} from './constants';
 import {MDCTextFieldAdapter} from './adapter';
 import MDCTextFieldFoundation from './foundation';
-import {MDCTextFieldBottomLine} from './bottom-line';
-import {MDCTextFieldHelperText} from './helper-text';
+import {MDCTextFieldBottomLine, MDCTextFieldBottomLineFoundation} from './bottom-line';
+import {MDCTextFieldHelperText, MDCTextFieldHelperTextFoundation} from './helper-text';
```

Comment:
Same as above. Adding `// eslint-disable-next-line no-unused-vars` ensures only specific lines are allowed to ignore rule